.github/workflows/generate-sdks.yml

```diff
@@ -2,6 +2,31 @@ name: Generate SDKs
 
 on:
   workflow_dispatch:
+    inputs:
+      manual_sdk_branch:
```

Comment:
shouldn't need to have to specify this, but 'calculate' i from a constant and the `version_postfix`

Reply:
Replaced with concatination release-manual-{{postfix}}